docs/security.rst

```diff
@@ -402,3 +402,22 @@ not set.
 
     [core]
     default_impersonation = airflow
+
+
+Flower Authentication
+---------------------
+
+Basic authentication for Celery Flower is supported.
+
+You can specify the details either as an optional argument in the Flower process launching
+command, or as a configuration item in your ``airflow.cfg``. For both cases, please provide
+`user:password` pairs separated by a comma.
+
+.. code-block:: bash
+
+    airflow flower --basic_auth=user1:password1,user2:password2
```

Comment:
The CLI option should probably be avoided as it would show up in `ps` output.

Is it also the raw password, or is it the salted/hashed output from `htpasswrd` or similar?

Reply:
In addition, actually both options provided here (specifying the Flower auth details as an CLI argument, or a configuration item in `airflow.cfg`) will expose the details in `ps` output, given eventually the Flower process is launched by `os.execvp` (https://github.com/apache/incubator-airflow/blob/6097f829ac5a4442180018ed56fa1b695badb131/airflow/bin/cli.py#L1299-L1300)

But as shared, this is how this Flower basic authentication works (while implementing OAuth for Flower in Airflow scenario is not really necessary. Not mentioning the Flower in Airflow is "streaking" at this moment)

Reply:
Fair point.